cc3d/core/GraphicsOffScreen/MVCDrawViewBase.py

```diff
@@ -37,12 +37,35 @@ def __init__(self, _drawModel):
         self.__initDist = 0  # initial camera distance - used in zoom functions
         self.min_max_text_actor = vtk.vtkTextActor()
 
-
         # CUSTOM ACTORS
         self.customActors = {}  # {visName: CustomActorsStorage() }
         self.currentCustomVisName = ''  # stores name of the current custom visualization
         self.currentVisName = ''  # stores name of the current visualization
         self.cameraSettingsDict = {}  # {fieldName:CameraSettings()}
+        self.ren = ren
+
+    @property
+    def drawModel(self):
+        return self._drawModel()
+
+    @drawModel.setter
+    def drawModel(self, _i):
+        self._drawModel = ref(_i)
+
+    @property
+    def ren(self):
```

Comment:
do we need ren method in both Model and View classes? Perhaps we do but if we could keep it in only one of the classes it would be better I think . Not sure what the right answer here is so just posing a question